src/UI/components/upload/data/Upform.tsx

```diff
@@ -47,8 +48,16 @@ function Upform() {
 
   return (
     <form>
+      <TemplateDownload />
```

Comment:
I'm not sure about having this above the form, especially as the accordion is gone... I think people will download the template once, then not care about it in subsequent uploads, so we don't want it taking up too much space or attention.

Reply:
Ray wanted it to move down the page like a story. So he wanted the accordions removed and the templates at the top. So, one goes to upload - they grab the template they want/need - fill it in - upload (moving down the page)

Reply:
Ah, fine